### DISCUSSION TITLE
Help with accessing local server at home from vpn

### DISCUSSION THREAD

**Discussion by mobihen** [wg-easy/wg-easy#725] [Q&A]
Hi :)

FIrst of all, great work!

I want to be able to use my mobile internet connection to connect to my server apps at home.
My server at home is located at: 192.168.31.2, and to access portainer, it's on port 9000
No matter what I did, I cannot access there.. when I try to brows to 192.168.31.2:9000 after activating the vpn, it's loading for ever. 

Also, I've opened the ports on my router and forward it to 192.168.31.2
TCP   51821	192.168.31.2
UDP  51820	192.168.31.2
Please advise what am I missing here..?

Here is my config on docker file:

```dockerfile
version: "3.8"
volumes:
  etc_wireguard:

services:
  wg-easy:
    environment:
      # ⚠️ Required:
      # Change this to your host's public address
      # (example)
      - WG_HOST=170.231.154.60

       #Optional:
      - PASSWORD=admin
      - WG_PORT=51820
      - WG_PERSISTENT_KEEPALIVE=25
      - WG_DEFAULT_ADDRESS=70.60.87.x
      - WG_DEFAULT_DNS=1.1.1.1, 8.8.4.4
      - WG_MTU=1420
      - WG_ALLOWED_IPS=192.168.31.0/24
      - WG_DEVICE=wlp2s0 #I'm using wifi to connect my server to the network so this is the device found with ifconfig
      # - WG_PRE_UP=echo "Pre Up" > /etc/wireguard/pre-up.txt
      # - WG_POST_UP=echo "Post Up" > /etc/wireguard/post-up.txt
      # - WG_PRE_DOWN=echo "Pre Down" > /etc/wireguard/pre-down.txt
      # - WG_POST_DOWN=echo "Post Down" > /etc/wireguard/post-down.txt

    image: ghcr.io/wg-easy/wg-easy
    container_name: wg-easy
    volumes:
      - etc_wireguard:/etc/wireguard
    ports:
      - "51820:51820/udp"
      - "51821:51821/tcp"
    restart: unless-stopped
    cap_add:
      - NET_ADMIN
      - SYS_MODULE
    sysctls:
      - net.ipv4.ip_forward=1
      - net.ipv4.conf.all.src_valid_mark=1
```

**Comment by leesonaa** [comment]
the WG_ALLOWED_IPS means all of your client 
ip segment,if you set it as 0.0.0.0/0,it would be alloow any client

**Reply by mobihen** [reply]
I did try with 0.0.0.0/0

but nothing works.. 
should I use the up/down/etc scripts?..

**Reply by leesonaa** [reply]
@mobihen 
maybe the probelm is your WG_DEVICE
https://github.com/wg-easy/wg-easy?tab=readme-ov-file#options
author say it's worked by ehernet device not wlan device , you can change it and try.

**Reply by mobihen** [reply]
He said that the default is eth0.. 
Trying to put network mode host for the container

**Reply by leesonaa** [reply]
if you not defined the WG_DEVICE , the default interface is eth0 inside container,you can try  # - WG_DEVICE ,network_mode: bridge
